File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean valid = false;
        int j = 0;
        
        for(int i = 0; i < nums.length - 1; i++) {
            for(j = i + 1; j < nums.length; j++) {
                if(nums[i] == nums[j]) {
                    return true;
                }
            }
        }

        return false;
    }
}

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-1.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        max_heap = [-n for n in nums]
        heapq.heapify(max_heap)

        res = []
        heapq.heapify(res)
        
        while k > 0:
            value = heapq.heappop(max_heap)
            heapq.heappush(res, -value)
            k -= 1
        
        return res[0]